include/EventInserterPQ.h

```diff
@@ -0,0 +1,44 @@
+#ifndef EVENTINSERTERPQ_H
+#define EVENTINSERTERPQ_H
+
+#include <string>
+#include <vector>
+#include <map>
+#include <libpq-fe.h>
+//#include "Event.h"
+//#include "Root2Event.h"
+//#include "TupleReader.h"
+
+class EventInserterPQ{
+private:
+	//connector
+	PGconn* conn;
```

Comment:
data members end with an '_'. Example

PGconn *conn_; // instead of PGconn *conn